eve/workers/release/pod.yaml

```diff
@@ -0,0 +1,45 @@
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: release-worker
+spec:
+  containers:
+    - name: release-worker
+      image: "{{ images.release }}"
+      command: ["/bin/sh"]
+      args: ["-c", "buildbot-worker create-worker . ${BUILDMASTER}:${BUILDMASTER_PORT} ${WORKERNAME} ${WORKERPASS} && buildbot-worker start --nodaemon"]
+      resources:
+        requests:
+          cpu: "250m"
+          memory: 1Gi
+        limits:
+          cpu: "1"
+          memory: 1Gi
+      env:
+        - name: DOCKER_HOST
+          value: localhost:2375
+      volumeMounts:
+      - name: worker-workspace
+        mountPath: /home/eve/workspace
+    - name: dind-daemon
+      image: docker:18.06.1-dind
+      resources:
+        requests:
+          cpu: "500m"
+          memory: 1Gi
+        limits:
+          cpu: "1"
+          memory: 1Gi
```

Comment:
It was `2Gi` before, would we running into memory limits?

Reply:
I checked the metrics, and it is more than enough. The pods do consume more CPU then ram, but even that, the amount given is more than ok.